docs/components/modeler/desktop-modeler/process-applications.md

```diff
@@ -70,3 +72,15 @@ Any file within a process application can be linked as a resource. Linking a res
 - Manually by setting the process, decision, or form ID in the properties panel
 
 <p><img src={LinkResourcesImg} alt="Linking resources by using the replace feature" /></p>
+
+## Deploying a process application
+
+Process applications can be deployed using the [deploy feature](./connect-to-camunda-8.md). When deploying a process application, all files that are part of the process application will be deployed.
+
+<p><img src={DeployImg} alt="Deploying a process application" /></p>
+
+## Starting a process instance
+
+For any process of a process application an instance can be started using the [start instance feature](./start-instance.md). All process application files will be deployed automatically.
```

Comment:
We could emphasize the why: Start an instance in the state that you currently develop.

```suggestion
For any process of a process application an instance can be started using the [start instance feature](./start-instance.md). All process application files will be deployed automatically, ensuring you start the instance in the state that you currently develop.
```